scripts/deploy.sh

```diff
@@ -11,10 +11,10 @@ VERSION=$(grep -o '".*"' ./version/version.go | sed 's/"//g')
 run "operator-sdk build apirator/apirator:$VERSION"
 run "docker push apirator/apirator:$VERSION"
 
-DEPLOYMENT=$(kubectl get deployment -l app=apirator -n oas -o=name)
```

Comment:
Maybe we could pass the namespace as an argument to the deploy script. What do you think?